tests/ui/perfherder/alerts-view/alert_status_countdown_test.jsx

```diff
@@ -0,0 +1,560 @@
+  const dueDateIcon = await waitFor(() => getByTestId('triage-clock-icon'));
+
+  fireEvent.mouseOver(dueDateIcon);
+
+  const dueDateStatus = await waitFor(() => getByTestId('due-date-status'));
+  const dueDateStatusText = dueDateStatus.querySelector('p').innerHTML;
+  expect(dueDateStatusText).toBe('Triage: 2 days left');
+});
+
+test('Alert is created on Monday, Triage countdown shows 1 working days', async () => {
+  const alert = testAlertSummaries[0];
+
+  // created date day is set to Monday
+  alert.created = '2022-02-07T11:41:31.419156';
+  alert.triage_due_date = '2022-02-10T11:41:31.419156';
+
+  // current day is set to Wednesday
+  Date.now = jest.fn(() => Date.parse('2022-02-09T11:41:31.419156'));
+
+  const { getByTestId } = testStatusDropdown([], alert);
+  const dueDateIcon = await waitFor(() => getByTestId('triage-clock-icon'));
+
+  fireEvent.mouseOver(dueDateIcon);
+
+  const dueDateStatus = await waitFor(() => getByTestId('due-date-status'));
+  const dueDateStatusText = dueDateStatus.querySelector('p').innerHTML;
+  expect(dueDateStatusText).toBe('Triage: 1 days left');
+});
+
+test('Alert is created on Monday, Triage countdown shows hours left', async () => {
+  const alert = testAlertSummaries[0];
+
+  // created date day is set to Monday
+  alert.created = '2022-02-07T11:41:31.419156';
+  alert.triage_due_date = '2022-02-10T11:41:31.419156';
+
+  // current day is set to Thursday
+  Date.now = jest.fn(() => Date.parse('2022-02-10T01:41:31.419156'));
+
+  const { getByTestId } = testStatusDropdown([], alert);
+  const dueDateIcon = await waitFor(() => getByTestId('triage-clock-icon'));
+
+  fireEvent.mouseOver(dueDateIcon);
+
+  const dueDateStatus = await waitFor(() => getByTestId('due-date-status'));
+  const dueDateStatusText = dueDateStatus.querySelector('p').innerHTML;
+  expect(dueDateStatusText).toBe('Triage: 10 hours left');
+});
+
+test('Alert is created on Monday, Triage countdown shows Overdue', async () => {
+  const alert = testAlertSummaries[0];
+
+  // created date day is set to Monday
+  alert.created = '2022-02-07T11:41:31.419156';
+  alert.triage_due_date = '2022-02-10T11:41:31.419156';
+
+  // current day is set to Friday
+  Date.now = jest.fn(() => Date.parse('2022-02-11T11:41:31.419156'));
+
+  const { getByTestId } = testStatusDropdown([], alert);
+  const dueDateIcon = await waitFor(() => getByTestId('triage-clock-icon'));
+
+  fireEvent.mouseOver(dueDateIcon);
+
+  const dueDateStatus = await waitFor(() => getByTestId('due-date-status'));
+  const dueDateStatusText = dueDateStatus.querySelector('p').innerHTML;
+  expect(dueDateStatusText).toBe('Triage: Overdue');
+});
+
+test('Alert is created on Monday, Bug countdown shows 5 working days', async () => {
+  const alert = testAlertSummaries[0];
+
+  // created date day is set to Monday
+  alert.created = '2022-02-07T11:41:31.419156';
+  alert.first_triaged = '2022-02-07T11:41:31.419156';
+  alert.bug_due_date = '2022-02-14T11:41:31.419156';
```

Comment:
`bug_due_date` represents the 6th day from the created day?

Reply:
Shouldn't the 5th day be the last working day for the `bug_due_date`?
It seems to be a little confusing that the `bug_due_date` is set to be the next working day.

Reply:
`created = '2022-02-07T11:41:31.419156'` is Monday, 5 days would be on Saturday, but it's weekend, and the next working day is Monday:

- +1 is Tuesday
- +2 is Wednesday
- +3 is Thursday
- +4 is Friday
- +5 is Monday)
=> `bug_due_date = '2022-02-14T11:41:31.419156'`